components/artist_page_cta/view.coffee

```diff
@@ -53,10 +53,18 @@ module.exports = class ArtistPageCTAView extends Backbone.View
       afterAuthPath: @afterAuthPath
     @$(".artist-page-cta-overlay__register input[name='name']").focus()
     @$('.artist-page-cta-overlay__close').on 'click', @closeOverlay
+    @disableScroll()
+
+  disableScroll: ->
+    $('#main-layout-container').addClass('artist-page-cta__no-scroll')
```

Comment:
And yea, I didn't want to add any CSS for this in `main_layout` since it's only needed in the artist page asset package.